frontend/src/js/controllers/createChallengeUsingUi.js

```diff
@@ -0,0 +1,613 @@
+// Invoking IIFE for create challenge using ui
```

Comment:
Change the name of file to createChallengeUsingUiCtrl and be ensure to add 'Ctrl' at the end of each file of controller.